### PR TITLE
Enable DTS on webOS 23

### PR DESCRIPTION
**Pull request by Noltari**
**Changes**
Enables DTS playback on 2023 LG TVs (webOS 23).

**Issues**
Fixes https://github.com/jellyfin/jellyfin-webos/issues/172

### CODE REVIEW

File: src/scripts/browser.js
Comment (dmitrylyzo) [inline src/scripts/browser.js:120]: Isn't it called `23`?

https://webostv.developer.lge.com/develop/specifications/web-api-and-web-engine

Also, please add webOS 22 (`browser.versionMajor >= 87`).

Comment (Noltari) [reply src/scripts/browser.js]: Done in 47c86301a1be056307afb53b3739447190adee2d

File: src/scripts/browserDeviceProfile.js
Comment (dmitrylyzo) [inline src/scripts/browserDeviceProfile.js:505]:
```suggestion
        if (browser.tizenVersion >= 4 || (browser.web0sVersion >= 5 && browser.web0sVersion < 23)) {
```

**Review by dmitrylyzo — APPROVED** [review]
LGTM

**Comment by Noltari** [comment]
> LGTM, commits can be squashed.

Commits squashed in f04d7e6af0855d4f09fef611c9da54b5de05b334

**Comment by Noltari** [comment]
@dmitrylyzo since this is my first contribution to jellyfin I'm not familiar with the development process, so... Is there anything else needed for this to be merged?
It would be great to have it on the next jellyfin release :)
(And excuse me for mentioning you but I don't know where to ask this...)
Thanks in advance!

**Comment by dmitrylyzo** [comment]
> @dmitrylyzo since this is my first contribution to jellyfin I'm not familiar with the development process, so... Is there anything else needed for this to be merged? It would be great to have it on the next jellyfin release :) (And excuse me for mentioning you but I don't know where to ask this...) Thanks in advance!

We are waiting for someone from the core team for final review.
I've added this PR to the `Release Future` board so it won't be missed.

**Pull request merged**
